docs/install/binary.mdx

```diff
@@ -46,6 +46,18 @@ If your platform isn't available, you must
   a pull request to add it to this page!
 </Tip>
 
+### Arch Linux
+
+The latest tagged release of Ghostty is available as [ghostty](https://archlinux.org/packages/extra/x86_64/ghostty/) in Arch Linux's `[extra]` repository.
+Installation is just like any other package:
+
+```console
+$ pacman -S ghostty
```

Comment:
The `$` character can get annoying for those who click the copy-to-clipboard button